tests/Exercism.TestRunner.FSharp.IntegrationTests/DirectoryHelper.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Exercism.TestRunner.FSharp.IntegrationTests
```

Comment:
Great to have this working. I was wondering if you'd be interested in converting all the C# code to F#? If not, we can merge this PR as is and convert to F# later.

Reply:
I would suggest merging this PR. 

I will convert the code to F# later.

Reply:
Sure, will do!

Reply:
Works for me!